test/e2e/framework/globalegressip.go

```diff
@@ -0,0 +1,116 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package framework
```

Comment:
To fix linting error...

```suggestion
*/

package framework
```

Reply:
@tpantelis After submitting this PR, as discussed during our meeting, I was thinking if we should keep this file in Submariner repo or in Shipyard repo. What are your thoughts?

Reply:
Anyway, let me update it and we can keep the file in Shipyard as we have other Globalnet related files in the same folder.

Reply:
Yeah I thought we decided to keep it in submariner since other projects don't need it and likely won't.  We can always move it later if needed.

Reply:
Let us keep it in Shipyard as clusterGlobalEgressIP and GlobalIngressIP files are also in Shipyard. Hope you are fine with it.